client/me/purchases/controller.jsx

```diff
@@ -101,23 +100,21 @@ export default {
 		);
 	},
 
-	confirmCancelPurchase( context ) {
+	confirmCancelDomain( context ) {
 		setTitle(
-			titles.confirmCancelPurchase
+			titles.confirmCancelDomain
 		);
 
 		recordPageView(
-			paths.confirmCancelPurchase(),
-			'Confirm Cancel Purchase'
+			paths.confirmCancelDomain(),
+			'Confirm Cancel Domain'
```

Comment:
We could yeah, is it clearer what information is sent to the analytic engine though?